### DISCUSSION TITLE
Win 11 突然無法打中文

### DISCUSSION THREAD

**Discussion by Jentheworld** [rime/home#1372] [Q&A]
如上，剛剛下載時還能夠打中文，但是過幾日後就無反應，就算切換成中文模式都只能打英文
右鍵按不同選項，如輸入法設定或用戶詞典管理都沒有反應
嘗試了重新下載或者設定算法服務都沒有用，算法服務甚至說遺失捷徑，也無法在電腦上運行

**Comment by fxliang** [comment]
看看最新的ci产物是否用好转？https://github.com/rime/weasel/actions

**Comment by hhtmp88** [comment]
Simply execute "小狼毫算法服務"

see this post：
weasel 0.15 + win10 突然無法輸入，也無法重新部署 #1018
2023.1101
[https://github.com/rime/weasel/issues/1018#issuecomment-1793924289](https://github.com/rime/weasel/issues/1018#issuecomment-1793924289)
